doc/test.md

```diff
@@ -77,20 +79,20 @@ You can enable coverage metrics for any npm command with the
 Examples:
 
 ```bash
-./scripts/coverage.sh npm run test
-./scripts/coverage.sh npm run test-unit
+./scripts/coverage.sh yarn test
+./scripts/coverage.sh yarn test-unit
```

Comment:
I don't know yarn so well, so I suppose `yarn test` works the same as `npm test`. But shouldn't this one be `yarn run test-unit`?